src/vs/editor/contrib/find/browser/simpleFindWidget.css

```diff
@@ -10,6 +10,8 @@
 	display: flex;
 	padding: 4px;
 	align-items: center;
+	width: 255px;
```

Comment:
Might be better to keep the default smaller for now since there's no clever adding padding to top like in the editor. The find result is also shown on the first line generally so the chance of a collision is fairly high.